File: python/pcats_api_example_manuscript_7.py
import pcats_api_client as pcats_api

#example 8

jobid=pcats_api.staticgp(datafile="../data/example7.csv",
        outcome="Y",
        treatment="A",
        x_explanatory="X",
        x_confounding="X",
        burn_num=500,
        mcmc_num=500,
        outcome_type="Continuous",
        tr_type="Discrete",
        method="GP",
        mi_datafile="../data/example7_midata.csv")

print("JobID: {}".format(jobid))

status=pcats_api.wait_for_result(jobid)

if status=="Done":
    print(pcats_api.printgp(jobid))
else:
    print("Error")

File: R/pcatsAPIclientR/pcats_api_example_manuscript_7.R
library(pcatsAPIclientR)

#example 8

jobid <- pcatsAPIclientR::staticGP(datafile="../../data/example7.csv",
        outcome="Y",
        treatment="A",
        x.explanatory="X",
        x.confounding="X",
        burn.num=500,
        mcmc.num=500,
        outcome.type="Continuous",
        method="GP",
        tr.type="Discrete",
        mi.datafile="../../data/example7_midata.csv")

cat(paste0("JobID: ",jobid,"\n"))

status <- pcatsAPIclientR::wait_for_result(jobid)

if (status=="Done") {
   cat(pcatsAPIclientR::printgp(jobid))
}

